crates/bevy_ecs/src/query/filter.rs

```diff
@@ -76,6 +76,7 @@ impl<T: Component> WorldQuery for With<T> {
 }
 
 /// The [`Fetch`] of [`With`].
+#[derive(Copy)]
 pub struct WithFetch<T> {
     marker: PhantomData<T>,
 }
```

Comment:
Using the derive here means that `WithFetch` will only be `Copy` if `T` is `Copy`.
For symmetry with `Clone`, `Copy` should be manually implemented without any bounds on `T`.

Reply:
I should've noticed that since deriving Clone would not work. Thank you for pointing it out.